feeds/wlan-ap/opensync/patches/32-add-new-manufacturing-data.patch

```diff
@@ -0,0 +1,277 @@
+Index: opensync-2.0.5.0/interfaces/opensync.ovsschema
+===================================================================
+--- opensync-2.0.5.0.orig/interfaces/opensync.ovsschema
++++ opensync-2.0.5.0/interfaces/opensync.ovsschema
+@@ -199,6 +199,66 @@
+               "type": "integer"
+             }
+           }
++        },
++        "qr_code": {
++          "type": {
++            "key": "string",
++            "value": {
++               "type": "string",
++               "maxLength": 128
++            },
++            "min": 0,
++            "max": "unlimited"
++          }
++        },
++        "model_description": {
++          "type": {
++            "key": {
++              "type": "string"
++            }
++          }
++        },
++        "manufacturer_name": {
++          "type": {
++            "key": {
++              "type": "string"
++            }
++          }
++        },
++        "manufacturer_date": {
++          "type": {
++            "key": {
++              "type": "string"
++            }
++          }
++        },
++        "manufacturer_url": {
++          "type": {
++            "key": {
++              "type": "string"
++            }
++          }
++        },
++        "reference_design": {
++          "type": {
++            "key": {
++              "type": "string"
++            }
++          }
++        },
++        "certification_region": {
++          "type": {
++            "key": {
++              "type": "string"
++            }
++          }
++        },
++        "id": {
```

Comment:
Just checking, this should replace the existing "id": column, correct?
i.e. you just removed the "min": 0, "max": 1

Reply:
Thanks for catching this. "id" was already defined, and I happen to redefine it without min/max. I did not see any failure/complaints by the build system though. I will remove this redefinition I made.